src/Codeception/Test/Loader/Cest.php

```diff
@@ -63,6 +63,10 @@ function ($v) {
                 if (!empty($dataMethod)) {
                     try {
                         $data = ReflectionHelper::invokePrivateMethod($unit, $dataMethod);
+                        if ($data instanceof \Traversable) {
```

Comment:
I'd propose doing this with a single loop and manual merge. 
Manual merging is faster than `array_merge` and when first doing `iterator_to_array` it will be even worse.

So, just remove the test and iterate:
```php
foreach($data as $xample)  {
$examples[] = $example;
}
```

Reply:
Nice idea, thanks. Did so in 05b891350edc564fb1617205b96b3f842f4530e3